.travis.yml

```diff
@@ -126,6 +126,8 @@ matrix:
       if: branch = auto
     - env: IMAGE=dist-armv7-linux DEPLOY=1
       if: branch = auto
+    - env: IMAGE=dist-cloudabi DEPLOY=1
+      if: branch = auto
```

Comment:
I don't think we want to put this as a separate job on the CI for now due to resource limitation.